_docs/setup/kubernetes/automatic-sidecar-inject.md

```diff
@@ -49,6 +49,9 @@ gcloud container clusters create NAME \
     --zone=ZONE
 ```
 
+### IBM Bluemix Container Service
+If your cluster is v1.7.4 or newer, you'll have the required [alpha feature](https://kubernetes.io/docs/admin/extensible-admission-controllers/#enable-initializers-alpha-feature) enabled by default.  
```

Comment:
Line 20 already has a link to the k8s alpha features.

Reply:
That is a link for what are initializers... this is a direct link for alpha feature enablement.